streamer/src/quic.rs

```diff
@@ -49,7 +49,7 @@ pub(crate) fn configure_server(
     let config = Arc::get_mut(&mut server_config.transport).unwrap();
 
     // QUIC_MAX_CONCURRENT_STREAMS doubled, which was found to improve reliability
-    const MAX_CONCURRENT_UNI_STREAMS: u32 = (QUIC_MAX_CONCURRENT_STREAMS * 2) as u32;
+    const MAX_CONCURRENT_UNI_STREAMS: u32 = (QUIC_MAX_UNSTAKED_CONCURRENT_STREAMS * 2) as u32;
```

Comment:
unrelated. i can't help but ask why this doubling is here rather than SDK where the variable is declared?

Reply:
@ryleung-solana can you please remove the doubling now that streams don't have head-of-line blocking? https://github.com/solana-labs/solana/pull/26086